aries-site/src/examples/templates/feedback/ModalTaskFlowExample/PowerOnFlow.js

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import { Anchor, Box, Button, Spinner, Text } from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+import { ButtonGroup } from '../../ButtonGroup';
+import { ModalBody, ModalFooter } from '../../ModalDialog';
+
+const steps = {
+  'confirm-power-on': {
+    body: (
+      <Box gap="small">
+        <Text size="large">You are powering on 1 device:</Text>
+        <Text size="large">FTCUSAMountain_ML350Gen10_testcluster</Text>
+      </Box>
+    ),
+    footer: ({ onCancel, onConfirm }) => (
+      <ButtonGroup>
+        <Button onClick={onCancel} label="Cancel" />
+        <Button
+          label="Power on 1 device"
+          onClick={onConfirm}
+          primary
+          type="submit"
+        />
+      </ButtonGroup>
+    ),
+  },
+  'powering-on': {
+    body: (
+      <Box gap="small">
+        <Text size="large">You are powering on 1 device:</Text>
+        <Spinner
+          alignSelf="center"
+          size="small"
+          message={{
+            start: 'Powering on devices.',
+            end: 'Power on attempt complete.',
```

Comment:
It would be nice if the "end" message was a little more specific than "complete" -- is there any way to know the success/failure of the attempt?

Reply:
In the real-world scenario, I believe this flow is initiating the power on signal, kicking off an asynchronous process. Success would be delivered in the form of a notification. 

I'm not sure if mocking a success check is worth the effort / additional complexity for the purpose of this example.

Reply:
That's fair. I'd say let's leave as is for now.

Reply:
agree